packages/syft/src/syft/service/worker/worker_pool.py

```diff
@@ -43,6 +65,7 @@ class SyftWorker(SyftObject):
     image_hash: Optional[str]
     healthcheck: Optional[WorkerHealth]
     status: WorkerStatus
+    full_image_tag: Optional[str]
```

Comment:
MAYBE: instead of full_image_tag we could have it as image_tag?

Reply:
MAYBE: For image_tag, we could pass SyftWorkerImageTag class from SyftWorkerImage

Reply:
Done 
![image](https://github.com/OpenMined/PySyft/assets/88959106/c5964610-5018-42e0-8299-f1cd878f48a2)